### DISCUSSION TITLE
Значения в output

### DISCUSSION THREAD

**Discussion by arthur-s** [snakers4/silero-vad#109] [Q&A] [answered]
Добрый день, пробую onnx модель, и после запуска session.run (для 4000 элементного сэмпла) в аутпуте получаю два значения, например 0.94601 и 0.0567758 для первого сэмпла файла files_ru.wav. Я так понимаю, первый параметр это вероятность того, что сэмпл это речь, верно? А что представляет второе значение?

**Comment by snakers4** [comment, marked as answer]
>  в аутпуте получаю два значения, например 0.94601 и 0.0567758 

Тот факт, что в сумме они примерно равны 100, намекает, что это два логита сетки после софтмакса, а во всех примерах используется первый (нулевой), потому что там - логит для речи.

В обширных примерах для ONNX указано какой и как использовать - https://colab.research.google.com/github/snakers4/silero-vad/blob/master/silero-vad.ipynb :


![image](https://user-images.githubusercontent.com/12515440/137672634-4b9e2277-5e7e-4dd2-aff9-e5ce2b34ef2a.png)

**Reply by arthur-s** [reply]
Понял, спасибо!
